docs/src/app/page.tsx

```diff
@@ -44,7 +44,7 @@ const HeroSection = () => {
       <div className="relative z-10 text-center">
         <h1 className="text-4xl sm:text-5xl md:text-6xl font-bold font-mono mb-4">bulloak</h1>
         <p className="text-lg sm:text-xl mb-8">
-          A test generator based on the Branching Tree Technique
+          A Solidity test generator based on the Branching Tree Technique
```

Comment:
It's no longer only `Solidity`! There's also [poinciana](https://github.com/ericnordelo/poinciana), which generates Cairo tests.

Reply:
Oh wow! I didn't know that.